s3cortxutils/s3confstore/ut/s3confstore_unit_tests.py

```diff
@@ -0,0 +1,79 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+#!/usr/bin/python3.6
+
+import mock
+import unittest
+import json
+import os.path
+from s3confstore.cortx_s3_confstore import S3CortxConfStore
+from cortx.utils.conf_store import Conf
+
+class S3ConfStoreAPIsUT(unittest.TestCase):
+  """UT to test current APIs"""
+
+  def test_json_conf(self):
```

Comment:
also test empty config path argument to load_config function.

Now we can remove uttest method from the source python class.

Also write UT for set config